mmv1/products/compute/ResourcePolicy.yaml

```diff
@@ -334,3 +345,22 @@ properties:
         name: 'expirationTime'
         description: |
           The expiration time of the schedule. The timestamp is an RFC3339 string.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'diskConsistencyGroupPolicy'
+    min_version: 'beta'
+    conflicts:
+      - 'snapshot_schedule_policy'
+      - 'group_placement_policy'
+      - 'instance_schedule_policy'
+    description: |
+      Replication consistency group for asynchronous disk replication.
+    send_empty_value: true
```

Comment:
why `send_empty_value` only to then use the encoder to delete it? Is it due to the case of empty block? 

Then we should simply remove the default of the field `enabled` and make it required.

Reply:
I have made it required now and removed both send_empty_value and default_value.

Reply:
Yes, this is because combined with the custom expander (which removes the `enabled` flag and just sends an empty object), it generates this:

```
v, ok := d.GetOkExists("disk_consistency_group_policy"); !isEmptyValue(reflect.ValueOf(diskConsistencyGroupPolicyProp)) && (ok || !reflect.DeepEqual(v, diskConsistencyGroupPolicyProp)) {
```

`isEmptyValue` determines that the empty map generated is in fact empty (Len() == 0) and does not send it.